backend/api/question.py

```diff
@@ -0,0 +1,31 @@
+"""This API Route handles serving the questions and documentation"""
+
+from fastapi import APIRouter, Depends, HTTPException
+from ..services.exceptions import InvalidCredentialsException
+from ..models import QuestionsPublic, Team
+from ..services.questions import QuestionService
+from ..services.auth import AuthService
+from .auth import authed_team
+
+__authors__ = ["Nicholas Almy"]
+
+openapi_tags = {
+    "name": "Questions",
+    "description": "Routes for Question retrieval",
+}
+
+api = APIRouter(prefix="/api/questions")
+
+
+@api.get("/", response_model=QuestionsPublic, tags=["Questions"])
```

Comment:
Nit: For consistency, I would prefer we use `/api/questions` over `/api/questions/` (the trailing slash is usually omitted)